components/grid/grid.html

```diff
@@ -0,0 +1,188 @@
+            <a href="../avatar/avatar.html" class="sidebar-link">Avatar</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../badge/badge.html" class="sidebar-link">Badge</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../button/button.html" class="sidebar-link">Button</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../card/card.html" class="sidebar-link">Card</a>
+          </li>
+          <li
+            class="sidebar-item mp-m active-component"
+            style="background-color: var(--grid-color)"
+          >
+            <a href="./grid.html" class="sidebar-link">Grid</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../image/image.html" class="sidebar-link">Image</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../input/input.html" class="sidebar-link">Input</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../list/list.html" class="sidebar-link">List</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../modal/modal.html" class="sidebar-link">Modal</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../navigation/navigation.html" class="sidebar-link"
+              >Navigation</a
+            >
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../rating/rating.html" class="sidebar-link">Rating</a>
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../text-utilities/text-utilities.html" class="sidebar-link"
+              >Text Utilities</a
+            >
+          </li>
+          <li class="sidebar-item mp-m">
+            <a href="../toast/toast.html" class="sidebar-link">Toast</a>
+          </li>
+        </ul>
+      </aside>
+      <section class="doc-section mp-m">
+        <h2 class="doc-header font-xl">Grid</h2>
+        <p class="doc-text mpy-s">
+          CSS Grid is a two-dimensional grid system used to work on the layout
+          of UI elements and segments of a webpage. The Grid comprises
+          horizontal and vertical lines to form rows and columns, much like a
+          table.
+        </p>
+
+        <!-- || Grid 20-80 Layout -->
+        <div
+          class="component-type mp-s"
+          style="border-color: var(--grid-color)"
+        >
+          <h3 class="component-header font-l">Grid 20-80 Layout</h3>
+          <p class="component-text mpy-s">
+            This layout renders the 20:80 ratio parent containers.
+          </p>
+          <div class="component-display mp-m">
+            <div class="grid grid-20-80 gap">
+              <div style="border: 2px dashed var(--secondary-color)">
+                Grid I
+              </div>
+              <div style="border: 2px dashed var(--secondary-color)">
+                Grid II
```

Comment:
if possible ,you can remove inline style same in line 95

Reply:
Yeah that is because i want user could remove it. It shouldn't be part of my library.